build-swift-benchmarks-with-swiftpm.txt

```diff
@@ -0,0 +1,14 @@
+// Test that we can build the Swift benchmarks using SwiftPM.
+//
+// REQUIRES: have-swift-benchmarks
+//
+// RUN: rm -rf %t.dir
+// RUN: mkdir -p %t.dir/
+//
+// RUN: %{swift} build --package-path %{swift_benchmarks_path} --build-path %t.dir 2>&1 | tee %t.build-log
+//
+// Check the build log.
+//
+// RUN: %{FileCheck} --check-prefix CHECK-BUILD-LOG --input-file %t.build-log %s
+//
+// CHECK-BUILD-LOG: Linking{{.*}} {{.*}}SwiftBench
```

Comment:
Actually, can you add to this an invocation. Specifically:

```
./swiftbench --num-iters=1 --tags=
```

Basically just run each test with 1 iteration or something like that to make sure everything works.

Reply:
I am also fine if you just run 1 test. I would use XorLoop. So:

```
./SwiftBench --num-iters=1 XorLoop
```

Just make sure it exits 0.